tools/topology/sof-apl-src-50khz-pcm512x.m4

```diff
@@ -0,0 +1,72 @@
+#
+# Topology for Apollolake UP^2 with pcm512x codec with sample rate
```

Comment:
@lgirdwood I think this was request from @plbossart at some point... the reason it is in this PR is that this PR makes it work, it was not possible to run it before these changes. So it is here for both "feature request" and for testing. However, It can be made as another PR or dropped if not needed anymore.